readme/contribution/CONTRIBUTING.md

```diff
@@ -57,15 +57,24 @@ Connected to Test Cluster at 127.0.0.1:9042.
 Use HELP for help.
 cqlsh>
 ```
+>If you are running Cadence on top of [Mysql](MYSQL_SETUP.md) or [Postgres](POSTGRES_SETUP.md), you can follow the instructions to run the SQL DB and then run:
+
 Now you can setup the database schema
 ```bash
 make install-schema
+
+# If you use SQL DB, then run:
+make install-schema-mysql OR make install-schema-postgres
 ```
 
 Then you will be able to run a basic local Cadence server for development:
 ```bash
 ./cadence-server start
+
+# If you use SQL DB, then run:
+./cadence-server --zone <mysql/postgres> start
```

Comment:
Isn't it development_mysql/development_postgres?

Reply:
I think the loading logic will automatically append the value for `zone` to `environment` when resolving the config file path.